### ISSUE TITLE
Set translucent background for description?

### ISSUE THREAD

**Issue by basememara** [ealeksandrov/EAIntroView#118]
Great framework, thx!! Is there a way to add a translucent background to the description label? I have high-res background images and the description sometimes camouflages against the background image in some places, so I would like to place a translucent background color to the description so the text shows no matter how crazy the background image is. Is this possible without having to create a custom intro hopefully?

**Comment by ealeksandrov** [comment]
How about adding blurring or fading whole background image? I can add options to IntroView instance, so it will fade and/or blur its background. Will it work for you?

**Comment by basememara** [comment]
Yes, that was actually another option that crossed my mind! Blurring would look real elegant and the fading would ensure words would show up in various scenarios I think.

**Comment by ealeksandrov** [comment]
Great, will add this options on this week. Most likely it will be included in 3.0.0 release.

**Comment by basememara** [comment]
Awesome, thx again for an amazing framework! BTW, you made my top 10 CocoaPods list :)
http://basememara.com/top-10-swift-friendly-cocoapods/

**Comment by ealeksandrov** [comment]
Awesome, thanks for mentioning :)
